packages/hub/main.ts

```diff
@@ -218,6 +220,10 @@ export async function bootWorker() {
         let task = await container.instantiate(PersistOffChainPrepaidCardCustomizationTask);
         return task.perform(payload, helpers);
       },
+      'persist-off-chain-merchant-info': async (payload: any, helpers: Helpers) => {
+        let task = await container.instantiate(PersistOffChainMerchantInfoTask);
+        return task.perform(payload, helpers);
+      },
```

Comment:
In the category of boilerplate cleanup, we could likely add a macro-like function to generate methods of this type.

Reply:
Good suggestion. I'll add a new ticket and address this later so that we can merge this and unblock several other tickets that depend on this PR.